ANSWERS.md

```diff
@@ -0,0 +1,23 @@
+1. Proptypes are used to check the form of data a component is 
+    given. This way, developers will be notified if they have
+    passed down the wrong data to a component and change it 
+    immediately instead of them spending time trying to find
+    what they did wrong
+
+2.  A life-cycle event is one of the events in a react component's 
+    lifecycle. They could be one of [mount, update, and unmount], 
+    developers can choose to do something when these events occur.
+
+3.  Higher Order Components are used to reuse stateful logic to
+    other components. It takes in a component as a prop and returns
+    a component with a stateful logic that the HOC has passed down 
+    to the component prop.
+
```

Comment:
Yes, good answer. It should be noted that a Higher Order Component is a function and not technically a component in and of itself. Although, you could characterize it as a functional component but that's not really the case.